main.go

```diff
@@ -140,11 +141,20 @@ func main() {
 		registry = prometheus.DefaultRegisterer.(*prometheus.Registry)
 	}
 
+	passwordMap := make(map[string]string)
+	if *redisPwd == "" && *redisPwdFile != "" {
+		passwordMap, err = exporter.LoadPwdFile(*redisPwdFile)
+		if err != nil {
+			log.Fatalf("Use empty redis password map.")
```

Comment:
You could improve this error messgae by saying something like:

`log.Fatalf("Error loading redis passwords from file %s, err: %s", *redisPwdFile, err)`